CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -713,7 +700,7 @@ local function CreateSettingsIcon(topBarInstance)
 			settingsActive = false
 		end
 
-		MenuModule:ToggleVisibility(settingsActive)
+		MenuModule:ToggleVisibility(settingsActive, nil, nil, usingGamepad)
```

Comment:
I am a little confused here, I see `MenuModule:SetVisibility` had 4 params, but `MenuModule:ToggleVisibility` appears to have only 1.